packages/p2p/src/components/floating-rate/floating-rate.tsx

```diff
@@ -36,10 +36,17 @@ const FloatingRate = ({
         ui: { current_focus, setCurrentFocus },
     } = useStore();
 
+    const { handleSubscription, exchange_rates } = useExchangeRate2();
+
+    React.useEffect(() => {
+        handleSubscription('USD', local_currency);
+    }, [local_currency, handleSubscription]);
+
+    const exchange_rate = exchange_rates.USD[local_currency] || 1;
+
```

Comment:
just a question, since a lot of this is repeated through different components, would it be alright to create a hook inside p2p to return the exchange rate? we can also pass the local currency to that hook, since some we pass through props and others we pass the currency config

Reply:
I agree. @niloofar-deriv do you think it will take much time? if so we can proceed with this since the BE is pending to this card and later we can create another pr for enhancement. 
is that ok ?

Reply:
Sure will update that